Platforms/QemuQ35Pkg/Test/PlatformTest.py

```diff
@@ -0,0 +1,246 @@
+from collections import namedtuple

```

Comment:
Normally we add a copyright header to Python files.

Reply:
Good catch. I've added this to both files. Thanks!